+mystica/nomenclature.md

```diff
@@ -0,0 +1,57 @@
+# Nomenclature
+## Variables
```

Comment:
Is this document linked or mentioned somewhere? Furthermore, I would document here the software necessary to install to render LaTeX formulas in GitHub Markdown files.

Reply:
> Is this document linked or mentioned somewhere?

No! I added a pointer in the readme
https://github.com/ami-iit/mystica/pull/1/commits/394bc767a13d922b7ef002adcc960d90afce7207#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R82-R83

> Furthermore, I would document here the software necessary to install to render LaTeX formulas in GitHub Markdown files.

You are right!
https://github.com/ami-iit/mystica/pull/1/commits/394bc767a13d922b7ef002adcc960d90afce7207#diff-30f1590c9d25c9533cc089fcdfed706a2b237024223e18ca16db6d857b2de656R4-R5